CMP1903_A1_2324/Game.cs

```diff
@@ -6,16 +6,36 @@
 
 namespace CMP1903_A1_2324
 {
+    /// <summary>
+    /// This creates the dice roll.
+    /// </summary>
     internal class Game
     {
-        /*
-         * The Game class should create three die objects, roll them, sum and report the total of the three dice rolls.
-         *
-         * EXTRA: For extra requirements (these aren't required though), the dice rolls could be managed so that the
-         * rolls could be continous, and the totals and other statistics could be summarised for example.
-         */
+        private static Die newDice = new Die();
+        private static Testing debug = new Testing();
 
-        //Methods
+        /// <summary>
+        /// This gathers each die's number and calculates the sum of them all.
+        /// </summary>
+        public void game()
+        {
+            // Creates variables for the loop.
+            int i = 1;
+            var list = new List<int> { };
+            int j = 3;
+            while (i <= j)
+            {
+                int rollDice = newDice.Roll();
+                Console.WriteLine($"Dice " + i + ": " + rollDice);
+                debug.Assert(rollDice);
```

Comment:
Some comments could be added to help explain and improve readability within the while loop.

Reply:
Added 👍🏻